src/main/java/org/onebusaway/alexa/exception/OneBusAwayException.java

```diff
@@ -0,0 +1,17 @@
+package org.onebusaway.alexa.exception;
+
+/**
+ * OneBusAway exception will thrown when Alexa handler failed to process the request.
```

Comment:
`will be thrown when the Alexa handler fails to process the request.`

Reply:
Done